_pages/security/incident-response-drills.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Incident Response Drills
+sidenav: security
+sticky_sidenav: true
+---
+
+*Table of Contents*
+
+* [Why do Incident Response Drills?](#why-do-incident-response-drills)
+* [How to Build Incident Response Drills](#how-to-run-an-incident-response-drill)
+* [Example Incident Response Drills](#example-incident-response-drills)
+* [Using this drill as part of your ATO](#congratulations-you-accidentally-did-compliance-too)
+
+## Why do Incident Response Drills?
+
+You don't want to be creating or testing recovery processes while things are on fire. 🔥
+
+Preparing and practicing ahead of time is a good idea. Running incident response drills on an annual basis at the very least is a good idea!
+
+## How to Run an Incident Response Drill
+
+### Identify Your Top Risks
+
+First, create a boundary diagram if you don't already have one. You will very likely need to create a boundary diagram as part of your system's security and compliance process, anyway.
```

Comment:
Good catch!

Reply:
```suggestion
First, create a [boundary diagram](https://www.fedramp.gov/determining-your-fedramp-boundary-definition/) if you don't already have one. You will very likely need to create a boundary diagram as part of your system's security and compliance process, anyway.
```

Reply:
☝️ This was the best I could find from a gov source, if anyone finds a better reference or definition feel free to swap out the link!

Reply:
I think the cloud.gov examples are more helpful for the task at hand: https://cloud.gov/docs/compliance/diagrams/ 

I would probably also just call it a system diagram. That isn't a huge deal for me.

Reply:
Yeah, might be scope creep to go too deep into system diagrams / boundary diagrams since our focus on this page is incident response. Those cloud.gov examples do look helpful though!